packages/webapp/src/components/Noun/Noun.module.css

```diff
@@ -82,9 +93,9 @@
 }
 
 @media only screen and (min-width: 1025px) {
-    .img {
-        width:  500px;
-        max-width: 500px;
+    /* Single & Double Noun */
+    .nounWrapper .img, .nounsWrapper .img, {
+        width: 500px;
     }
 
     .nounId {
```

Comment:
Do you think the Noun ID display will work out? Not critical but might be able to cut some stuff if not!